doc/source/dev/conduct/code_of_conduct.rst

```diff
@@ -119,7 +119,7 @@ scipy-conduct@googlegroups.com. Currently, the committee consists of:
 - Ralf Gommers
 
 If your report involves any members of the committee, or if they feel they have
-a conflict of interest in handling it, then they will recuse themselves from
+a conflict of interest in handling it, then they will recurse themselves from
```

Comment:
recuse is a correct word. It should not be diagnosed at all. If necessary, switch the "rare" dictionary that you mentioned in cython to "no action"

Reply:
It is diagnosed by the _rare_ dictionary because in the context of source code it is more often a typo of `recurse` than the `recuse` itself. But that's what the _rare_ dictionary is for.